python/ray/util/client_connect.py

```diff
@@ -25,6 +28,10 @@ def connect(
     ray_init_kwargs: Optional[Dict[str, Any]] = None
 ) -> Dict[str, Any]:
     if ray.is_connected():
+        ignore_reinit_error = ray_init_kwargs.get("ignore_reinit_error", False)
+        if ignore_reinit_error:
+            logger.info("Calling ray.init() again after it has already been called.")
```

Comment:
May be a bit more specific, e.g. reusing the same Ray client connection because ray.init() has been called?

Also, should this return the same object returned by the first `ray.init()` call? It may be available in the Ray client context. If that is too hard, we can mention that the connection info is available from the first ray.init() call in the log message above too.